### DISCUSSION TITLE
f951: Fatal Error: Reading module ‘noahmpvartype’ at line 1605 column 40: Unexpected EOF

### DISCUSSION THREAD

**Discussion by 12wlj** [NCAR/hrldas#177] [Q&A]
我在编译noahmp时，在设置完user_build_options后，make后出现以下问题，求大神帮忙解决：
/usr/bin/cpp -P -traditional -D_GFORTRAN_  ConfigVarInitMod.F90 > ConfigVarInitMod.f90
/usr/bin/gfortran -c -I. -I../utility -I../drivers/hrldas -g -fconvert=big-endian -fbounds-check -fno-range-check  -ffree-form  -ffree-line-length-none -I/home/zhanghd/Build_WRF_1/LIBRARIES_2/netcdf/include ConfigVarInitMod.f90
f951: Fatal Error: Reading module ‘noahmpvartype’ at line 1605 column 40: Unexpected EOF
compilation terminated.
make[1]: *** [Makefile:149: ConfigVarInitMod.o] Error 1
make[1]: Leaving directory '/home/zhanghd/hrldas-master/noahmp/src'
make: *** [Makefile:10: all] Error 2

**Comment by cenlinhe** [comment]
Hi, this problem occurred because something is wrong with your computer machine environment setup for compiler. Please refer to this discussion thread: https://github.com/NCAR/hrldas/discussions/171
You may want to consult with your IT staff to solve the problem, or use our NoahMP docker container mentioned in the above discussion thread.
